DDD-12.md

```diff
@@ -0,0 +1,157 @@
+# Quarkus Debezium Extension
+
+Quarkus Debezium Extension should provide a simple way to receive data source events inside a Quarkus Native Application and apply some logic to them (see image) thanks to the engine and an appropriate connector.
+
+![](./DDD-12/dbz-emb.png)
+
+In order to be able to receive those events from the data-source, the actual way to achieve it is to import the `debezium-engine` and a connector like in this way:
+
+```xml
+<dependency>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-embedded</artifactId>
+    <version>${version.debezium}</version>
+</dependency>
+<dependency>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-connector-mysql</artifactId>
+    <version>${version.debezium}</version>
+</dependency>
+```
+This approach doesn't work out-of-the-box in situations in which [we want to build a native image of the application](https://debezium.io/blog/2025/03/12/superfast-debezium/).
+
+## 1. Two dependency (Debezium quarkus engine and connector separate)
+The first solution is similar to the actual way to use it inside an application beside the fact that the dependencies are wrapped into quarkus extensions like:
+
+```xml
+<dependency>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-quarkus-embedded</artifactId>
+    <version>${version.debezium}</version>
+</dependency>
+<dependency>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-quarkus-connector-mysql</artifactId>
+    <version>${version.debezium}</version>
+</dependency>
+```
+
+![](./DDD-12/s1-dbz-lib.png)
+
+## 2. One dependency (Debezium quarkus engine with Quarkus connector)
+The second solution propose a unique dependency that contains the `engine` and the `connector` like in this way:
+
+```xml
+<dependency>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-quarkus-mysql</artifactId>
+    <version>${version.debezium}</version>
+</dependency>
+```
+
+the configuration property delegated to define the connector class should be unavailable and already defined inside the extension:
+```txt
+connector.class=io.debezium.connector.mysql.MySqlConnector
+```
+
+![](./DDD-12/s2-dbz-lib.png)
+
+## Quarkus Debezium Extension configuration
```

Comment:
Can you provide an example configuration for the extension? Those it have a prefix like `quarkus.debezium`?